src/make-server-run.ts

```diff
@@ -14,7 +14,7 @@ if (!module.parent) {
 		throw new Error("Must provide PORT");
 	}
 
-	const url = remote ? "http://types-publisher.azurewebsites.net" : `http://localhost:${port}`;
+	const url = remote ? "http://typespublisher.azurewebsites.net" : `http://localhost:${port}`;

```

Comment:
?

Reply:
We moved the server recently and the url changed.